module/Calibrator.cpp

```diff
@@ -101,23 +104,50 @@ void Calibrator::measureAndSetTargetBrightness()
   targetBrightness = (whiteBrightness + blackBrightness) / 2;
   snprintf(buf, SMALL_BUF_SIZE, ">> Target Brightness Value is %d", targetBrightness);
   logger.log(buf);
+
+  delete measurer;
 }
 
 void Calibrator::waitForStart()
 {
   char buf[SMALL_BUF_SIZE];  // log用にメッセージを一時保持する領域
   Logger logger;
   constexpr int startDistance = 5;  // 手などでスタート合図を出す距離[cm]
+  int count = 0;
+
+  Measurer* measurer = new Measurer();
 
   logger.log("On standby.\n");
   snprintf(buf, SMALL_BUF_SIZE, "On standby.\n\nSignal within %dcm from Sonar Sensor.",
            startDistance);
   logger.log(buf);
 
   // startDistance以内の距離に物体がない間待機する
-  while(measurer.getForwardDistance() > startDistance) {
-    timer.sleep();  // 10ミリ秒スリープ
+  // 1分毎にMeasurerインスタンスを再生する
+  while(1) {
+    while(measurer->getForwardDistance() > startDistance && count < 600) {
+      timer.sleep(100);  // 100ミリ秒スリープ
+      count++;
+      // 確認用（コマンドラインにセンサの取得値を出力）
+      if(count % 50 == 0) {
+        snprintf(buf, SMALL_BUF_SIZE, "count: %d\nbrightness: %d\nforwardDistance: %d", count,
+                 measurer->getBrightness(), measurer->getForwardDistance());
+        logger.log(buf);
+      }
+    }
+
+    if(count >= 600) {
+      // Measurerの再インスタンス化
+      delete measurer;
+      Measurer* measurer = new Measurer();
```

Comment:
deleteはポインタが指すメモリ領域を解放するだけで、ポインタ変数自体は消えないみたいなので、再代入する形にした方が良いということに今更気づいた、、、
こうするなら、メンバ変数としてmeasurerを宣言できる。
```suggestion
      measurer = new Measurer();
```